components/webfield/AuthorConsole.js

```diff
@@ -295,6 +295,192 @@ const AuthorConsoleTasks = () => {
   )
 }
 
+// #region config docs
+/** AuthorConsole config doc
+ *
+ * @typedef {Object} AuthorConsoleConfig
+ *
+ // eslint-disable-next-line max-len
+ * @property {Object} header mandatory but can be empty object
+ * @property {1|2} apiVersion mandatory
+ * @property {string} venueId mandatory
+ * @property {string} submissionId mandatory
+ * @property {string} authorSubmissionField mandatory
+ * @property {string} officialReviewName mandatory
+ * @property {string} decisionName=Decision optional
+ * @property {string|string[]|object[]} reviewRatingName mandatory
+ * @property {string} reviewConfidenceName mandatory
+ * @property {string} authorName mandatory
+ * @property {string} submissionName mandatory
+ * @property {boolean} showAuthorProfileStatus optional
+ * @property {string} blindSubmissionId optional
+ * @property {boolean} showIEEECopyright optional
+ * @property {string} IEEEPublicationTitle optional
+ * @property {number} IEEEArtSourceCode optional
+ */
+
+/**
+ * @name AuthorConsoleConfig.header
+ * @description Page header. Contains two string fields: "title" and "instructions" (markdown supported).
+ * @type {Object}
+ * @default no default value
+ * @example
+ * {
+ *   "header": {
+ *     "title": "Some conference",
+ *     "instructions": "some **instructions**"
+ *   }
+ * }
+ */
+
+/**
+ * @name AuthorConsoleConfig.apiVersion
+ * @description API version to request. use 2 for new venues.
+ * @type {1|2}
+ * @default no default value
+ * @example
+ * { "apiVersion": 2 }
+ */
+
+/**
+ * @name AuthorConsoleConfig.venueId
+ * @description The id of the venue group, usually should be value of domain.id
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "venueId": "ICLR.cc/202X/Conference" }
+ */
+
+/**
+ * @name AuthorConsoleConfig.submissionId
+ * @description The invitation id of submissions, used to get the notes for display in console.
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "submissionId": "ICLR.cc/20XX/Conference/-/Submission" }
+ */
+
+/**
+ * @name AuthorConsoleConfig.authorSubmissionField
+ * @description used to filter notes that current user is an author of.
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "authorSubmissionField": "content.authorids" }
+ */
+
+/**
+ * @name AuthorConsoleConfig.officialReviewName
+ * @description name of the offical review, used for header display and for composing the official review invitation id for filtering
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "officialReviewName": "Official_Review" }
+ */
+
+/**
+ * @name AuthorConsoleConfig.decisionName
+ * @description name of the decision note. used to construct decision invitation id to find the decision note and to display the paper decision.
```

Comment:
Name of the decision invitation, the decision invitation should have a field name "decision" to read and display the value